soc/xtensa/intel_adsp/common/ipc.c

```diff
@@ -98,7 +111,11 @@ void intel_adsp_ipc_complete(const struct device *dev)
 {
 	const struct intel_adsp_ipc_config *config = dev->config;
 
+#ifdef CONFIG_SOC_SERIES_INTEL_ACE1X
+	config->regs->tda = ACE_IPC_TDA1X_DONE;
```

Comment:
Again, should be replace with = 0 or ACK_TDA().

Reply:
Dropping ACE implementation in this PR. It waits too long for SOF MTL upstream.